tests/always/test_connection.py

```diff
@@ -275,13 +275,13 @@ def test_connection_extra_with_encryption_rotate_fernet_key(self):
             description="no schema",
         ),
         UriTestCaseConfig(
-            test_conn_uri="google-cloud-platform://?key_path=%2Fkeys%2Fkey.json&scope="
+            test_conn_uri="google-cloud-platform://@/?key_path=%2Fkeys%2Fkey.json&scope="
```

Comment:
Ideally I wouldn't update tests, but as far as I understand it doesn't comply with the CLI contract.